conu/apidefs/backend.py

```diff
@@ -63,7 +63,16 @@ def set_logging(
 
 class Backend(object):
     """
-    This class groups classes related to a specific backend.
+    This class groups classes and functionality related to a specific backend.
+
+    We strongly advise you to use backend as a context manager:
+
+    ::
+
+        with SomeBackend() as backend:
+            backend...
+
+    This will ensure that temporary directory associated with the backend instance will be cleaned.
```

Comment:
Reading this I was a bit confused what directory is this referring to. There might be also a little description of it and how it is used.